purchase_work_acceptance_late_fine/security/ir.model.access.csv

```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_work_acceptance_manager,access_work_acceptance_manager,model_work_acceptance,purchase.group_purchase_manager,1,1,1,1
```

Comment:
Do we need this again? I though we had it in module purchase_work_acceptance already.